src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5055,7 +5056,7 @@ private FieldDeclarationSyntax ParseConstantFieldDeclaration(SyntaxListBuilder<A
                 return _syntaxFactory.FieldDeclaration(
                     attributes,
                     modifiers.ToTokenList(),
-                    _syntaxFactory.VariableDeclaration(type, variables),
+                    _syntaxFactory.VariableDeclaration(type, variables, null),
```

Comment:
Are these changes necessary since we've kept the original `VariableDeclaration` overload?